docs/general/ad-filtering/create-own-filters.md

```diff
@@ -1486,6 +1488,10 @@ Use `@@` to negate `$removeheader`:
 >
 > 3. `$removeheader` rules are not compatible with any other modifiers except `$domain`, `$third-party`, `$app`, `$important`, `$match-case`, and [content type modifiers](#content-type-modifiers), e.g. `$script`, `$stylesheet`, etc. The rules which have any other modifiers are considered invalid and will be discarded.
 
+> **Note**
+>
+> When there are two rules: a blocking rule with `$header` - `||example.com^$header=value` and a modifying rule with `$removeheader` - `||example.com^$removeheader=value`, the engine will choose the blocking rule because the modifier `$header` adds 1 to the weight of the rule. For more information, go to [rules priorities](#rule-priorities).
```

Comment:
> the engine will choose the blocking rule because the modifier `$header` adds 1 to the weight of the rule

для корлибсов это некорректно, в них эти правила друг от друга не зависят

Reply:
удалил этот фрагмент